packages/core/src/components/templates/ButtonTemplates/SubmitButton.tsx

```diff
@@ -11,6 +11,7 @@ export default function SubmitButton<
   if (norender) {
     return null;
   }
+
```

Comment:
I would suggest just reverting this change since it really isn't necessary to fixing the bug